build/tasks/nodeify.js

```diff
@@ -10,7 +10,9 @@ module.exports = function (grunt) {
 				return grunt.file.read(fn);
 			}).join('\n');
 
-			grunt.file.write(fileset.dest, 'exports.source = ' + JSON.stringify(source) + ';');
+			var file = grunt.file.read(fileset.dest);
+
+			grunt.file.write(fileset.dest, file + 'module.exports.source = ' + JSON.stringify(source) + ';');
```

Comment:
Doesn't seem right indeed. How about instead we write the code to a different file, and have the current dest be a file that does two things: First it requires axe-core as a node module, and second it reads out that file so it can also expose the source.